### PR TITLE
Support feature to set supply on NFT

### PR DESCRIPTION
**Pull request by dvargasfuertes**
https://app.clarity.so/cabin/view/3039c279-2ee2-4da2-a604-dc1c23d5010c

### CODE REVIEW

File: packages/app/pages/checkout/[network]/[address].tsx [hunk 1]
```diff
@@ -258,7 +258,7 @@ export const getStaticPaths: GetStaticPaths<QueryParams> = () => {
   });
 };
 
-export const getStaticProps: GetStaticProps<PageProps, QueryParams> = (
+export const getServerSideProps: GetServerSideProps<PageProps, QueryParams> = (
```
Comment (dvargasfuertes) [inline packages/app/pages/checkout/[network]/[address].tsx:261]: fixes a bug where the supply on checkout page would be stale

File: packages/app/pages/index.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import Image from "next/image";
 import React, { useCallback, useEffect, useMemo, useState } from "react";
 import { Contract, ContractSendMethod } from "web3-eth-contract";
 import { TransactionReceipt } from "web3-core";
-import { Modal, Input, Button, Label } from "@cabindao/topo";
+import { Modal, Input, Button, Label, Toast } from "@cabindao/topo";
```
Comment (dvargasfuertes) [inline packages/app/pages/index.tsx:9]: Also wanted to introduce the new Toast component in this PR

File: packages/contracts/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cabindao/nft-passport-contracts",
-  "version": "0.6.2",
+  "version": "0.6.3",
```
Comment (dvargasfuertes) [inline packages/contracts/package.json:3]: will import latest version into `/app` once this PR goes through

**Comment by dvargasfuertes** [comment]
tagging @ssaitejar96  for review

**Review by ssaitejar96 — APPROVED** [review]
Very nice!

**Pull request merged**